j/jquery-mousewheel/jquery-mousewheel_ubi_8.sh

```diff
@@ -0,0 +1,97 @@
+#!/bin/bash -e
+# -----------------------------------------------------------------------------
+#
+# Package		: jquery-mousewheel
+# Version		: 3.1.11
+# Source repo	: https://github.com/jquery/jquery-mousewheel.git
+# Tested on		: UBI 8.4
+# Language      : Node
+# Travis-Check  : True
+# Script License: Apache License, Version 2 or later
+# Maintainer	: Vaishnavi Patil <Vaishnavi.Patil3@ibm.com>
+#
+# Disclaimer: This script has been tested in root mode on given
+# ==========  platform using the mentioned version of the package.
+#             It may not work as expected with newer versions of the
+#             package and/or distribution. In such case, please
+#             contact "Maintainer" of this script.
+#
+# ----------------------------------------------------------------------------
+
+PACKAGE_NAME=jquery-mousewheel
+PACKAGE_VERSION=${1:-3.1.11}
+PACKAGE_URL=https://github.com/jquery/jquery-mousewheel.git
+
+set -ex
```

Comment:
not needed , we are already doing the same thing on line #1068

Reply:
Removed it from build-script